README.md

```diff
@@ -1,25 +1,38 @@
+[![CircleCI (all branches)](https://img.shields.io/circleci/project/github/glotzerlab/hoomd-blue.svg?style=flat)](https://circleci.com/gh/glotzerlab/hoomd-blue)
+[![Read the Docs](https://img.shields.io/readthedocs/hoomd-blue/stable.svg)](https://hoomd-blue.readthedocs.io/en/stable/?badge=stable)
+[![Contributors](https://img.shields.io/github/contributors/glotzerlab/hoomd-blue.svg?style=flat)](https://hoomd-blue.readthedocs.io/en/stable/credits.html)
+[![License](https://img.shields.io/badge/license-BSD--3--Clause-green.svg)](LICENSE)
+
+**HOOMD-blue** is a general purpose particle simulation toolkit.It performs hard particle Monte Carlo simulations
 of a variety of shape classes, and molecular dynamics simulations of particles with a range of pair, bond, angle,
-and other potentials. HOOMD-blue runs fast on NVIDIA GPUs, and can scale across
-many nodes. For more information, see the [HOOMD-blue website](http://glotzerlab.engin.umich.edu/hoomd-blue).
+and other potentials. **HOOMD-blue** runs fast on NVIDIA GPUs, and can scale across thousands of nodes.
+For more information, see the [**HOOMD-blue** website](https://glotzerlab.engin.umich.edu/hoomd-blue/).
 
-# Tutorial
+## Resources
 
-[Read the HOOMD-blue tutorial online](http://nbviewer.jupyter.org/github/joaander/hoomd-examples/blob/master/index.ipynb).
+- [Reference Documentation](https://hoomd-blue.readthedocs.io/):
+  Full package Python API, usage information, and feature reference.
+- [hoomd-users Google Group](https://groups.google.com/d/forum/hoomd-users):
+  Ask questions to the **HOOMD-blue** community.
+- [**HOOMD-blue** Tutorial](https://nbviewer.jupyter.org/github/glotzerlab/hoomd-examples/blob/master/index.ipynb):
+  Beginner's guide, code examples, and sample scripts.
+- [**HOOMD-blue** website](https://glotzerlab.engin.umich.edu/hoomd-blue/):
+  Additional information, benchmarks, and publications.
 
-## Installing HOOMD-blue
+## Installation
 
-**HOOMD-blue** binary images are available on [Docker Hub](https://hub.docker.com/) and packages on [conda-forge](https://conda-forge.org/).
+**HOOMD-blue** binaries are available via the
+[Docker image glotzerlab/software](https://hub.docker.com/r/glotzerlab/software)
+and for Linux and macOS via the
+[hoomd package on conda-forge](https://anaconda.org/conda-forge/hoomd). See links for more information.
 
-### Docker images
-
-Pull the [glotzerlab/software](https://hub.docker.com/r/glotzerlab/software/) to get **HOOMD-blue** along with
-many other tools commonly used in simulation and analysis workflows. Use these images to execute HOOMD-blue in
-Docker/Singularity containers on Mac, Linux, and cloud systems you control and on HPC clusters with Singularity support.
-CUDA and MPI operate with native performance on supported HPC systems
-See full usage information on the [glotzerlab/software docker hub page](https://hub.docker.com/r/glotzerlab/software/).
+### Using the Docker image
```

Comment:
``umask 002`` is no longer required. The singularity pull command should be: ``singularity pull --name "software.simg" shub://glotzerlab/software``